fiboa_cli/datasets/es_cat.py

```diff
@@ -0,0 +1,67 @@
+from ..convert_utils import convert as convert_
+import pandas as pd
+
+SOURCES = {
+    "https://analisi.transparenciacatalunya.cat/api/views/yh94-j2n9/files/b4299961-52ee-4fa0-a276-4594c8c094bc?download=true&filename=Cultius_DUN2023_GPKG.zip": ["Cultius_DUN2023_GPKG/CULTIUS_DUN2023.gpkg"]
+}
+ID = "es_cat"
+SHORT_NAME = "Catalonia"
+TITLE = "Catalonia Crop Fields (Mapa de cultius)"
+DESCRIPTION = """
+The Department of Agriculture, Livestock, Fisheries and Food makes available to the public the data from the crop map of Catalonia.
+This map allows you to locate the crops declared in the Agrarian Declaration - DUN submitted to the DACC.
+"""
+PROVIDERS = [
+    {
+        "name": "Catalonia Department of Agriculture, Livestock, Fisheries and Food",
+        "url": "https://agricultura.gencat.cat/ca/ambits/desenvolupament-rural/sigpac/mapa-cultius/",
+        "roles": ["producer", "licensor"]
+    }
+]
+ATTRIBUTION = "Catalonia Department of Agriculture, Livestock, Fisheries and Food"
+LICENSE = {
+    "title": "The Open Information Use License - Catalonia",
+    "href": "https://administraciodigital.gencat.cat/ca/dades/dades-obertes/informacio-practica/llicencies/",
+    "type": "text/html",
+    "rel": "license"
+}
+COLUMNS = {
+    "geometry": "geometry",
+    "id": "id",
+    "campanya": "determination_datetime",
+    "ha": "area",
+    "cultiu": "crop_name",
+}
+
+COLUMN_MIGRATIONS = {
+    "campanya": lambda col: pd.to_datetime(col, format='%Y'),
+    "geometry": lambda col: col.make_valid(),
```

Comment:
The question for explode_multipolygon is, do we always want "normal" polygons, or do we also allow multipolygons in our datasets? I always disliked multipolygons for crop fields/parcels, but that might be my bias.

And invalid polygons give you problems everywhere and all the time. I want to get rid of invalidity as early as possible...

Reply:
My thinking was that in convert we keep the source geometries as they are and then allow people to fix things when needed. If we want consistency, the alternative would be to always run make_valid and explode_multipolygon to ensure consistent datasets. Right now it seems pretty author specific, which options are used in the converters, which is not ideal.

Reply:
@cholmes I think you said at some point that converters should do less and we should push more into additional CLI commands? Is that also true for this case?

Reply:
One of the ideas behind collecting all these datasets is to make it really easy to use them. If all data is consistent valid and to a lesser degree also polygon, you don't have to fix data yourself. Given the documentation (data-survey and converter), someone can go back to the original file if required.

But it depends what we want to create.

Reply:
I'd say for source coop deployment we should indeed run the improve command, because otherwise files may be invalid actually. For me converters and the actual deployment are two different topics.

I'm thinking about FTW, if they may struggle to correctly judge/select examples if we modified some of the geometries. It would give them the option to themselves create original geometries fiboa compliant, but we can still run improve for deployment.

Reply:
=> Moved to #119